pkg/scheduler/metrics/job.go

```diff
@@ -37,8 +39,104 @@ var (
 			Help:      "Number of retry counts for one job",
 		}, []string{"job_id"},
 	)
+
+	jobPendingPhaseCount = promauto.NewGauge(
```

Comment:
IMO, we only need to pay attention to the final state, and we don’t care about other states because the state will change.